seawatch_registration/urls.py

```diff
@@ -20,7 +19,7 @@
     path('documents/<int:document_id>/edit', document.UpdateView.as_view(), name='document_update'),
     path('documents/<int:document_id>/delete', document.DeleteView.as_view(), name='document_delete'),
     path('documents/<int:document_id>/<str:file_name>', document.GetDocumentAttachment.as_view(), name='document_attachment_get'),
-    path('questions/edit/', question.UpdateView.as_view(), name='question_update'),
+    path('questions/edit/', question.UpdateView.as_view(), name='question_answer'),
```

Comment:
I think you need to update the RegirationCompleteView as well. (See my previous commits).

Reply:
i will update it in RegistrationCompleteView